src/NumericalAlgorithms/LinearOperators/Mass.hpp

```diff
@@ -0,0 +1,30 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+/// \file
+/// Defines function mass.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+
+/// \cond
+class DataVector;
+template <size_t>
+class Index;
+/// \endcond
+
+/*!
+ * \ingroup NumericalAlgorithmsGroup
+ * \brief Applies the mass matrix.
+ *
+ * \details We use a diagonal mass matrix approximation. Please refer to
+ * dg::lift_flux() for details.
+ */
```

Comment:
Can you please expand this comment to say more about what this code is doing? Something along the lines of what you put in the PR, e.g. "A linear operator that applies the mass matrix to a DataVector in the diagonal mass matrix approximation. This is needed to make the operator in elliptic equations symmetric, which helps with their iterative solution. For details on the diagonal mass matrix approximation, please see spectre/src/NumericalAlgorithms/DiscontinuousGalerkin/LiftFlux.hpp." It's fine to refer to other comments, but I'd like to be able to know what the function does when I first look at the hpp file.